docs/stdlib/array.rst

```diff
@@ -161,14 +161,17 @@ Reference
 
 .. eql:operator:: arrayslice: array<anytype> [ int64 : int64 ] -> anytype
 
-    Array slicing.
+    Slices an array of :eql:type:`anytype` containing a fixed range of
+    :eql:type:`int64` into a representable reference of the array's elements.
 
-    An omitted lower bound defaults to zero, and an omitted upper
-    bound defaults to the size of the array.
+    Omitting the lower bound an array will default the result to zero.
+    Doing so to the upper bound will also default to the current size of the
+    array.
 
-    The upper bound is non-inclusive.
+    The upper bound an array is non-inclusive.
```

Comment:
I can't parse the "an array" part.

Reply:
Typo, thank you for catching, should be "... of an array" or "... of the array"